crates/holochain_types/src/dht_op.rs

```diff
@@ -72,7 +72,7 @@ pub enum DhtOp {
     ///
     /// - Ensure that *the action alone* passes surface-level validation.
     /// - Store the action into their DHT shard.
-    ///   - FIXME: @artbrock, do they?
+    //   - FIXME: @artbrock, do they?
```

Comment:
Should this be included in docs or is it a code comment? I don't love seeing this kind of thing on docs.rs :D

Reply:
Same here, why would I put such reminders in the docs?!